src/main/java/org/broadinstitute/hellbender/utils/test/SamAssertionUtils.java

```diff
@@ -215,6 +257,32 @@ private static String equalHeadersIgnoreCOandPG(final File actualSam, final File
             return msg;
         }
     }
+    private static String equalHeadersIgnoreCOandPG(final Path actualSam, final Path expectedSam, final ValidationStringency validation, final Path reference) throws IOException {
```

Comment:
Can you make the file overload call into this one?

Reply:
Oh actually I meant to delete it; it's private and has no callers.